perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_group_attribute_def_virt_isGroupAdmin.java

```diff
@@ -0,0 +1,64 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.*;
+import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.MemberNotExistsException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
+import cz.metacentrum.perun.core.api.exceptions.WrongReferenceAttributeValueException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import cz.metacentrum.perun.core.implApi.modules.attributes.MemberGroupVirtualAttributesModuleAbstract;
+import cz.metacentrum.perun.core.implApi.modules.attributes.MemberGroupVirtualAttributesModuleImplApi;
+
+import java.util.List;
+
+
+/**
+ * @author Kristyna Kysela
+ */
+public class urn_perun_member_group_attribute_def_virt_isGroupAdmin extends MemberGroupVirtualAttributesModuleAbstract implements MemberGroupVirtualAttributesModuleImplApi {
+
+    public Attribute getAttributeValue(PerunSessionImpl sess, Member member, Group group, AttributeDefinition attributeDefinition) throws InternalErrorException {
+
+        List<User> userAdminList;
+        User newUser;
+        Attribute attribute = new Attribute(attributeDefinition);
+
+        try{
+            userAdminList = sess.getPerunBl().getGroupsManagerBl().getAdmins(sess, group, true);
+            newUser = sess.getPerunBl().getUsersManagerBl().getUserByMember(sess, member);
+        }catch (NullPointerException ex){
```

Comment:
There is no need to catch NullPointerException anywhere in perun-core.  (Also, you should never pass null as an argument to any method in core unless it's specifically allowed in documentation.)